app/controllers/categories_controller.rb

```diff
@@ -73,6 +73,8 @@ def show
     if Category.topic_create_allowed(guardian).where(id: @category.id).exists?
       @category.permission = CategoryGroup.permission_types[:full]
     end
+    @category.notification_level = CategoryUser.where(category: @category.id).first.notification_level
```

Comment:
You'll have to make sure that the `CategoryUser` belongs to the `current_user` as well. :smile: